File: src/test/java/kitchenpos/ordertable/OrderTableFixture.java

package kitchenpos.ordertable;

import kitchenpos.common.vo.Name;
import kitchenpos.ordertable.domain.OrderTable;
import kitchenpos.ordertable.vo.NumberOfGuests;

import java.util.UUID;

public class OrderTableFixture {

    public static OrderTable orderTable() {
        return new OrderTable(UUID.randomUUID(), new Name("주문테이블", false), new NumberOfGuests(1));
    }
}

File: src/test/kotlin/kitchenpos/ordertable/OrderTableFixture.kt
package kitchenpos.ordertable

import kitchenpos.domain.OrderTable

class OrderTableFixture {
    companion object {
        fun fixture(
            name: String = "테이블",
            numberOfGuests: Int = 0,
            occupied: Boolean = false
        ): OrderTable {
            val orderTable = OrderTable()
            orderTable.name = name
            orderTable.numberOfGuests = numberOfGuests
            orderTable.isOccupied = occupied
            return orderTable
        }
    }
}
